core/src/test/java/tech/pegasys/eth2signer/core/multikey/metadata/YamlSignerParserTest.java

```diff
@@ -105,8 +104,7 @@ void unencryptedMetaDataInfoWithInvalidHexEncodingForPrivateKeyFails() throws IO
 
     assertThatThrownBy(() -> signerParser.parse(filename))
         .isInstanceOf(SigningMetadataException.class)
-        .hasMessageStartingWith(
-            "Invalid signing metadata file format: Invalid hex value for private key");
+        .hasMessageStartingWith("Invalid signing metadata file format");
```

Comment:
Is this detail included somewhere else now? Do we need to remove it? I remember we went to the trouble of getting this appearing nice in the exception message.

Reply:
I'll create a test at the Integration Level to show these errors still appear nicely - but they are constructed in the DirectoryBackedSignerProvider - not the YamlSigner